main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Consider how this while loop is behaving, could we use a for loop, or need a loop at all? Consider a function and a helper function in this scenario.